### PR TITLE
amp-font: fix font loading promise resolved(safari)

### PR DESCRIPTION
**Pull request by zhouyx**
#4168
This is cause by a racing condition of failing to load font resource and first measurement of the font div height. 
By providing two standard font as default to font-family should solve this.

### CODE REVIEW

File: extensions/amp-font/0.1/fontloader.js [hunk 1]
```diff
@@ -217,7 +217,7 @@ export class FontLoader {
     const customFontElement = this.customFontElement_ =
         this.document_.createElement('div');
     style.setStyles(customFontElement, {
-      fontFamily: this.fontConfig_.family,
+      fontFamily: this.fontConfig_.family + ' ,san-serif, serif',
```
Comment (sriramkrish85) [inline extensions/amp-font/0.1/fontloader.js]: use

`this.fontConfig_.family + ',' + DEFAULT_FONTS_.join(',')`

https://github.com/zhouyx/amphtml/blob/0ef32eaf6b3a23fcfc826d2516b42ec82d0d3efd/extensions/amp-font/0.1/fontloader.js#L31

Comment (zhouyx) [inline extensions/amp-font/0.1/fontloader.js]: fixed

**Comment by zhouyx** [comment]
Disabled useNativeApis and tested in both chrome and safari.

File: extensions/amp-font/0.1/fontloader.js [hunk 2]
```diff
@@ -217,7 +217,7 @@ export class FontLoader {
     const customFontElement = this.customFontElement_ =
         this.document_.createElement('div');
     style.setStyles(customFontElement, {
-      fontFamily: this.fontConfig_.family,
+      fontFamily: this.fontConfig_.family + ',' + DEFAULT_FONTS_.join(','),
```
Comment (sriramkrish85) [inline extensions/amp-font/0.1/fontloader.js]: nit: you can just do join() and drop the ',' as that is the default

**Pull request merged**